### PR TITLE
DROID-2551 Membership | Active Builder on different accounts

### PR DESCRIPTION
**Pull request by konstantiniiv**
---

- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [x] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: payments/build.gradle [hunk 1]
```diff
@@ -47,6 +47,8 @@ dependencies {
 
     implementation libs.timber
 
+    implementation libs.gson
```
Comment (uburoiubu) [inline payments/build.gradle:50]: idem

File: payments/src/test/java/com/anytypeio/anytype/payments/MembershipTestsSetup.kt [hunk 2]
```diff
@@ -97,6 +105,16 @@ open class MembershipTestsSetup {
     @Before
     open fun setUp() {
         MockitoAnnotations.openMocks(this)
+        getAccount.stub {
+            onBlocking { async(Unit) } doReturn Resultat.success(
+                Account(
```
Comment (uburoiubu) [inline payments/src/test/java/com/anytypeio/anytype/payments/MembershipTestsSetup.kt:110]: StubAccount function can be used here

**Review by uburoiubu — APPROVED** [review]

File: app/src/main/java/com/anytypeio/anytype/di/feature/membership/MembershipDI.kt [hunk 3]
```diff
@@ -92,6 +94,12 @@ object MembershipModule {
         dispatchers: AppCoroutineDispatchers
     ): VerifyMembershipEmailCode = VerifyMembershipEmailCode(repo = repo, dispatchers = dispatchers)
 
+    @JvmStatic
+    @Provides
+    @PerScreen
+    fun provideGson(
```
[the remark this answers is not in the document]
Comment (konstantiniiv) [reply app/src/main/java/com/anytypeio/anytype/di/feature/membership/MembershipDI.kt]: updated

File: payments/build.gradle (hunk 1 above)
Comment (konstantiniiv) [reply payments/build.gradle]: done

File: payments/src/test/java/com/anytypeio/anytype/payments/MembershipTestsSetup.kt (hunk 2 above)
Comment (konstantiniiv) [reply payments/src/test/java/com/anytypeio/anytype/payments/MembershipTestsSetup.kt]: done

**Pull request merged**